builder-base/Dockerfile

```diff
@@ -1,140 +0,0 @@
-FROM centos:7
```

Comment:
is generated [here](https://github.com/jenkins-x/jenkins-x-builders/pull/131/files#diff-e091688d80187842c1b91dcc19485fbaR40)